httpaf-lwt.opam

```diff
@@ -0,0 +1,19 @@
+opam-version: "2.0"
+name: "httpaf-lwt"
+maintainer: "Spiros Eliopoulos <spiros@inhabitedtype.com>"
+authors: [ "Anton Bachin <antonbachin@yahoo.com>" ]
+license: "BSD-3-clause"
+homepage: "https://github.com/inhabitedtype/httpaf"
+bug-reports: "https://github.com/inhabitedtype/httpaf/issues"
+dev-repo: "git+https://github.com/inhabitedtype/httpaf.git"
+build: [
+  ["jbuilder" "subst" "-p" name] {pinned}
+  ["jbuilder" "build" "-p" name "-j" jobs]
+]
+depends: [
+  "ocaml" {>= "4.03.0"}
+  "faraday-lwt-unix"
+  "httpaf"
+  "jbuilder" {build & >= "1.0+beta10"}
+  "lwt"
+]
```

Comment:
I think this will fail the opam 2 lint check with `error 57: Synopsis and description must not be both empty`

Reply:
I started seeing the error after updating to opam 2.0.1 today